systemvm/patches/debian/buildsystemvm.sh

```diff
@@ -375,8 +375,10 @@ packages() {
   chroot . apt-get --no-install-recommends -q -y --force-yes install open-vm-tools
   #xenstore utils
   chroot . apt-get --no-install-recommends -q -y --force-yes install xenstore-utils libxenstore3.0
-  #keepalived and conntrackd
-  chroot . apt-get --no-install-recommends -q -y --force-yes install keepalived conntrackd ipvsadm libnetfilter-conntrack3 libnl1
+  #keepalived - install version 1.2.13 from wheezy backports
+  chroot . apt-get --no-install-recommends -q -y --force-yes -t wheezy-backports install keepalived
```

Comment:
Added the following not to the file and will push it to the PR:

#
# IMPORTANT REMARK
# Package intallation is no longer done via this script. We are not removing the code yet, but we want to 
# make sure that everybody willing to install/update packages should refer to the file:
#   ==> cloud-tools/appliance/definitions/systemvmtemplate/install_systemvm_packages.sh
#

Is it good enough?

Cheers,
Wilder